assets/pages/new-contributions-landing/components/ContributionThankYou/ContributionThankYou.scss

```diff
@@ -126,3 +104,21 @@
   padding-bottom: $gu-v-spacing;
 }
 
+.component-marketing-consent.component-marketing-consent--newsletter {
+  border-bottom: 1px solid gu-colour(garnett-neutral-4);
+}
+
+.component-marketing-consent__message {
+  font-family: $gu-text-egyptian-web;
+  letter-spacing: -0.5px;
+  margin-bottom: 24px;
```

Comment:
You could use ``$gu-v-spacing` * 2` here

Reply:
I copypasted the current code :D happy to do this in a subsequent PR

Reply:
Would that not be overkill? A whole PR to change `24px` to `$gu-v-spacing * 2`? Up to you, but I would recommend cleaning up as you go along here!